docs/source-app/workflows/byoc/index.rst

```diff
@@ -5,62 +5,287 @@ Run Apps on your own cloud (BYOC)
 
 **Audience:** Users looking to run Lightning Apps on their own private cloud.
 
+.. note::
+
+    This feature is in EARLY ACCESS. Contact us to request access.
+
 ----
 
-*******************
-A bit of background
-*******************
+***********************
+What is Lightning BYOC?
+***********************
 
 BYOC - Bring Your Own Cloud, is an alternate deployment model to Lightning Cloud (fully managed SaaS).
 BYOC separates the control and data plane. The data plane, that includes
 Lightning clusters, services and Lightning Apps, reside inside the user’s VPC.
+Lightning clusters, services and Lightning Apps (App), reside inside the user’s cloud account.
 The control plane resides on Lightning Cloud.
 
 Setup begins with configuring a cloud provider (today AWS, but more are coming soon) with your personal credentials for
 delegated access and an identity provider for secure access to the data plane.
+Setup begins with configuring a cloud provider (today AWS, but more are coming soon) with your
+personal credentials (this could include your CLI profile name, secret, and key) for delegated
+access and an identity provider for secure access to the data plane.
+
+After submitting a cluster creation request, the Lightning Control Plane creates the required cloud
+infrastructure on the user cloud account. On AWS this includes creating the EKS kubernetes cluster,
+necessary S3 buckets, IAM roles, and Lighting agent managing this environment.
+
+----
+
+***********************************
+What does Lightning BYOC do for me?
+***********************************
+
+Lightning BYOC allows you to use Lightning while keeping your data behind a wall.
+Lightning can interact with your data, but your data never leaves your cloud.
+This means you can run Apps, train Apps, collaborate on creating Apps and component
+within your own cloud. So your data is protected and you save money.
```

Comment:
the paragraph only elaborates data access - why does it save money?

Reply:
Updated